treescript/src/treescript/versionmanip.py

```diff
@@ -18,6 +18,7 @@
     "browser/config/version_display.txt",
     "config/milestone.txt",
     "mail/config/version.txt",
+    "mail/config/version_display.txt",
```

Comment:
This was wrongly deleted at https://github.com/mozilla-releng/scriptworker-scripts/pull/289/files#diff-ba2dbe092949999aa2d94bd9ec165f66c4b097d726845e3d85146392c81b1ca5L20